diff --git a/src/Relude.hs b/src/Relude.hs
index 7c5d7db..b0aa215 100644
--- a/src/Relude.hs
+++ b/src/Relude.hs
@@ -133,7 +133,49 @@ import Relude.Function
 import Relude.Functor
 import Relude.Lifted
 import Relude.List
-import Relude.Monad
+import Relude.Monad hiding
+  ( ExceptT (..)
+  , IdentityT (..)
+  , MaybeT (..)
+  , MonadReader (..)
+  , MonadState (..)
+  , Reader
+  , ReaderT (..)
+  , State
+  , StateT (..)
+  , ask
+  , asks
+  , evalState
+  , evalStateT
+  , evaluatingState
+  , evaluatingStateT
+  , exceptToMaybeT
+  , execState
+  , execStateT
+  , executingState
+  , executingStateT
+  , get
+  , gets
+  , hoistEither
+  , hoistMaybe
+  , local
+  , maybeToExceptT
+  , modify
+  , modify'
+  , put
+  , reader
+  , runExceptT
+  , runReader
+  , runState
+  , state
+  , usingReader
+  , usingReaderT
+  , usingState
+  , usingStateT
+  , withReader
+  , withReaderT
+  , withState
+  )
 import Relude.Monoid
 import Relude.Nub
 import Relude.Numeric
diff --git a/src/Relude/Monad/Trans.hs b/src/Relude/Monad/Trans.hs
index c2ac021..d1344cd 100644
--- a/src/Relude/Monad/Trans.hs
+++ b/src/Relude/Monad/Trans.hs
@@ -37,6 +37,7 @@ import Relude.Monad.Reexport (Either, ExceptT (..), Maybe, MaybeT (..), Reader,

 -- $setup
 -- >>> import Relude
+-- >>> import Relude.Monad

 {- | Shorter and more readable alias for @flip runReaderT@.

